### ISSUE TITLE
Prefer Usenet only until Usenet queue > configurable amount of hours

### ISSUE THREAD

**Issue by Apprisco** [Sonarr/Sonarr#6903]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Some of us like to use both usenet and torrents at the same time.
Usenet is cool and great and all, but when downloading 700GB shows it can get a little slow, even with 50MBps download speed. 
However, even when the usenet queue is at ~6 hours for the entirety of breaking bad, Sonarr prefers to use usenet over Torrents in delay profiles.

### Describe the solution you'd like

Sabnzbd and NZBGet both have API calls (that can be obtained with the same API key you put into download client configurations) that return current download speed and queue time.

For older shows as I have mentioned, the qualities and scores from recyclarr profiles tend to be exactly the same. As such, the delay profile is used to decide between Torrents and Usenet.

This queue time is important; In the delay profiles section, instead of just preferring Usenet then torrents, it'd be easy to add an option that lets you select # of time before getting the torrent instead.
In short:
Prefer Usenet until queue time > # of hours (configurable) then prefer Torrent.

### Describe alternatives you've considered

The alternative here is to just not use Torrents at all. Torrents are great though so it's really not a fun or useful option.
Instead of just having Torrents as a backup in case Usenet doesn't have the file I want, if the custom score is the same and the quality is the same, there should be a balancing profile instead of just preferring one or the other.

**Comment by thezoggy** [comment]
this already exists, called delay profiles. settings > profiles > delay profiles.

**Comment by markus101** [comment]
It's an interesting idea, but we do not have plans to get into such granular queue management at this time.
